src/modules/yield-farming/components/pool-rewards/index.tsx

```diff
@@ -41,6 +43,13 @@ const PoolRewards: React.FC = () => {
     return (sum ?? BigNumber.ZERO).plus(contract.potentialReward ?? BigNumber.ZERO);
   }, undefined);
 
+  const merkleDistributorData = yfPoolsCtx.merkleDistributor;
+  const isAirdropClaimed = merkleDistributorData?.isAirdropClaimed;
+  const adjustedAmount = merkleDistributorData?.adjustedAmount;
+
+  let airdropAmount;
+  !isAirdropClaimed ? (airdropAmount = BigNumber.from(adjustedAmount)) : BigNumber.from(0);
```

Comment:
May be it should be

`const airdropAmount = !isAirdropClaimed ? BigNumber.from(adjustedAmount) : BigNumber.from(0);`

Reply:
also fixed